cache_descriptor.go

```diff
@@ -72,27 +70,36 @@ func compare(old map[string]string, new map[string]string) (r result) {
 	return
 }
 
-// cacheDescriptor creates a cache descriptor for a given cache_path - change_indicator_path mapping.
-func cacheDescriptor(indicatorByCachePth map[string]string, method ChangeIndicator) (map[string]string, error) {
-	descriptor := map[string]string{}
-	for pth, indicatorPth := range indicatorByCachePth {
-		if len(indicatorPth) == 0 {
-			// this file's changes does not fluctuates existing cache invalidation
-			descriptor[pth] = "-"
-			continue
+// transformToIndicatorByPth transforms an indicator map to an indicatorByPth map.
+func transformToIndicatorByPth(indicatorMap map[string][]string) map[string]string {
```

Comment:
This is a duplicate of convertDescriptorToIndicatorByPath.

Reply:
Removed.